code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+	name = "Lonely Ritual"
+	gain_text = "I was able to combine my greed and desires to summon an eldritch beast i have not seen before."
+	desc = "You can now summon a Stalker using a knife, a flower, a pen and a piece of paper"
+	cost = 1
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/reagent_containers/food/snacks/grown/poppy,/obj/item/pen,/obj/item/paper)
+	mob_to_summon = /mob/living/simple_animal/hostile/eldritch/stalker
+	next_knowledge = list(/datum/eldritch_knowledge/summon/ashy,/datum/eldritch_knowledge/summon/rusty,/datum/eldritch_knowledge/flesh_final)
+	route = "Flesh"
+
+/datum/eldritch_knowledge/summon/ashy
+	name = "Ashen Ritual"
+	gain_text = "I combined principle of hunger with desire of destruction. The eyeful lords have noticed me."
+	desc = "You can now summon an Ash Man by transmutating a pile of ash , a head and a book."
+	cost = 1
+	required_atoms = list(/obj/effect/decal/cleanable/ash,/obj/item/bodypart/head,/obj/item/book)
+	mob_to_summon = /mob/living/simple_animal/hostile/eldritch/ash_spirit
+	next_knowledge = list(/datum/eldritch_knowledge/summon/stalker,/datum/eldritch_knowledge/spell/rust_wave)
+
+/datum/eldritch_knowledge/summon/rusty
+	name = "Rusted Ritual"
+	gain_text = "I combined principle of hunger with desire of corruption. The rusted hills call my name."
+	desc = "You can now summon a Rust Walker transmutating vomit pool, a head and a book."
+	cost = 1
+	required_atoms = list(/obj/effect/decal/cleanable/vomit,/obj/item/bodypart/head,/obj/item/book)
+	mob_to_summon = /mob/living/simple_animal/hostile/eldritch/rust_spirit
+	next_knowledge = list(/datum/eldritch_knowledge/summon/stalker,/datum/eldritch_knowledge/spell/mad_touch)
+
+/datum/eldritch_knowledge/spell/blood_siphon
+	name = "Blood Siphon"
+	gain_text = "Our blood is all the same after all, the owl told me."
+	desc = "You gain a spell that drains enemies health and restores yours."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/touch/ash_leech
+	next_knowledge = list(/datum/eldritch_knowledge/spell/rust_wave,/datum/eldritch_knowledge/spell/mad_touch)
+
+/datum/eldritch_knowledge/flesh_final
+	name = "Priest's Final Hymn"
+	gain_text = "Man of this world. Hear me! For the time of the lord of arms has come!"
+	desc = "Bring 3 bodies onto a transmutation rune to either ascend as a terror of the night prime or you can summon a regular terror of the night."
+	required_atoms = list(/mob/living/carbon/human)
+	cost = 3
+	route = "Flesh"
+	var/is_summoned = FALSE
+
+/datum/eldritch_knowledge/flesh_final/recipe_snowflake_check(list/atoms, loc)
+	if(is_summoned)
+		return FALSE
+	var/counter = 0
+	for(var/mob/living/carbon/human/H in atoms)
+		counter++
+	if(counter >= 3)
+		return TRUE
+	return FALSE
+
+/datum/eldritch_knowledge/flesh_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	var/alert_ = alert("Do you want to ascend as the lord of the night or just summon a terror of the night?","...","Yes","No")
```

Comment:
why is this a yes or no question?

Reply:
coz there are 2 options and i coded it at 2 am

Reply:
also another good catch i will have to fix it